backend/routes/currentState.js

```diff
@@ -34,6 +34,30 @@ router.post('/currentState', async (req, res) => {
       }
     }
 
+    // Retrieve active player information for the given location
+    // Check whether the user provided a last check timestamp
+    if (userLastCheckTime) {
+      // Retrieve the last update time for the given location
+      const locationSnapshot = await admin.firestore()
```

Comment:
You seem to have added the timestamp check again?

Reply:
Sorry - fixed. I think I made a mistake while merging.